pkgs/applications/misc/gxmessage/default.nix

```diff
@@ -0,0 +1,20 @@
+{stdenv, fetchurl, gnome3, intltool, pkgconfig, texinfo}:
+
+stdenv.mkDerivation rec {
+  name = "gxmessage";
+  version = "3.4.3";
+
```

Comment:
Don't worry too much about travis, the build is quite unreliable. You'll probably be asked to squash your commits, but other than that it looks good to merge to me.